app/uk/gov/hmrc/exports/repositories/RepositoryOps.scala

```diff
@@ -38,6 +43,10 @@ trait RepositoryOps[T] {
 
   val collection: MongoCollection[T]
 
+  // Field, if any, of the collection used to annotate the time, a java.time.Instant,
+  // of the last update (only, not for document insertions and replacements).
+  val lastUpdate: Option[String] = None
```

Comment:
Can I suggest you rename this val as 'lastUpdatedFieldName' to call out what it is holding (first glance I thought this field held a date/time value)